README.md

```diff
@@ -6,18 +6,26 @@ Install
 
 Instructions based on Ubuntu 14.04.
 
-Install the system package dependencies & virtualenv:
+If you're installing to your local Ubuntu 14.04:
 
 ```
-$ sudo apt-get install `cat debian_packages.lst`
-$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+$ make install_system_dependencies
 ```
 
-Ensure you load virtualenv with Python 3 in `~/.bashrc`:
+Or if you would prefer to use Vagrant:
+
+1. Install [Vagrant](https://www.vagrantup.com/) for your OS
+2. `$ vagrant up`
+3. `$ vagrant ssh`
+
+For either install method, continue with the following instructions:
+
+Install virtualenv and virtualenvwrapper for Python3
 
 ```
-export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
-source $HOME/.local/bin/virtualenvwrapper.sh
+$ make install_virtualenvwrapper_py3
+$ export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+$ source $HOME/.local/bin/virtualenvwrapper.sh
 ```
 
```

Comment:
@brousch Could also be worth adding a few lines about what to do to connect to the vagrant box (`vagrant ssh`) and get to the right directory (`cd opencraft`)